api/controllers/walletController.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Player = mongoose.model('Players'),
+    Transaction = mongoose.model('Transactions');
+
+exports.get = function(req, res) {
+    // todo check if has access
+    const playerId = req.params.playerId;
+    Player.find({playerId: playerId}).excec((err, player) => {
+        if(err) {
+            res.send(err);
+        }
+        res.json(player.wallet);
+    });
+};
+
+exports.transfer = function(req, res) {
+    // this should not be allowed to call by users
+    // this is either for top up account after winning bet, or top up made by cashier, or withdraw money

```

Comment:
Kasjer chyba nie ma możliwości podbicia portfela. W "okienku" użytkownicy są anonimami.

Reply:
Był pomysł, żeby wszystkie bet z okienka szły na konto kasjera - to byłby sposób na rozliczanie zmiany. A w tym przypadku myślałem, o czymś takim, że podchodzisz do okienka, żeby doładować konto gotówką, a nie złożyć bet. Możliwe, że taki use case jest w ogóle nie potrzebny

Reply:
Nie ma story z podbiciem walleta w okienku, więc myślę że możemy to olać. Swoją drogą, mamy też robić rozliczanie zmian?

Reply:
Jeśli te transakcje byłyby podpinane pod kasjera - co jest wygodne, bo mozna to tak samo obsługiwać, to wtedy mielibyśmy to prawie out of the box

Reply:
Ok, brzmi dobrze :)